src/main/java/reposense/git/GitLsTree.java

```diff
@@ -45,10 +49,8 @@ public static void validateFilePaths(RepoConfiguration config, Path clonedBareRe
 
         for (String path : paths) {
             path = StringsUtil.removeQuote(path);
-            Matcher matcher = ILLEGAL_WINDOWS_CHARACTER_PATTERN.matcher(path);
 
-            if (matcher.find()) {
-                logger.log(Level.SEVERE, String.format(MESSAGE_INVALID_PATH, path, matcher.group()));
+            if (!isValidWindowsFilename(path)) {
```

Comment:
Shouldn't we go through this test only when RepoSense is runned on windows?

Reply:
There’s already a check at the start of the method to see if the host is on Windows OS.

Reply:
OK